### PR TITLE
Patch config labels uppercase

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
### What changes were proposed in this pull request?
patch to accept uppercase letters labels

### Why are the changes needed?
Allow add uppercase letters as per kubernetes documentation is allowed

### Does this PR introduce any user-facing change?
yes

### How was this patch tested?
in a unit test on config_test.go

### CODE REVIEW

File: kubectl-cloudflow/config/config.go [hunk 1]
```diff
@@ -343,11 +343,11 @@ func labelHasPrefix(label string) bool {
 
 func validateLabel(name string, prefix string) error {
 	// See https://github.com/kubernetes/kubernetes/issues/71140, IsDNS1123Subdomain and IsDNS1123Label do not allow uppercase letters.
-	labelPattern := regexp.MustCompile(`^[a-z0-9]{1}[a-z0-9\.\_\-]{0,61}[a-z0-9]{1}$`)
+	labelPattern := regexp.MustCompile(`^[a-z0-9A-Z]{1}[a-z0-9\.\_\-]{0,61}[a-z0-9A-Z]{1}$`)
```
Comment (olofwalker) [inline kubectl-cloudflow/config/config.go:346]: Have k8s changed its stance on uppercase in names?

Comment (RayRoestenburg) [reply kubectl-cloudflow/config/config.go:346]: I have tried to add a label to a deployment with uppercase and it worked, so we should not prevent it.

Comment (RayRoestenburg) [reply kubectl-cloudflow/config/config.go:346]: (the docs say it is legal as well)

Comment (RayRoestenburg) [reply kubectl-cloudflow/config/config.go:346]: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set @olofwalker

**Review by RayRoestenburg — APPROVED** [review]

**Pull request merged**